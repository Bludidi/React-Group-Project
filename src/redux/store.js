import { configureStore } from '@reduxjs/toolkit';
import missionReducer from './missions/mission.redux';

export default configureStore({
  reducer: {
    mission: missionReducer,
  },
});
