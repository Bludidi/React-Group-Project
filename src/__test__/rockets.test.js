import { render } from '@testing-library/react';
import React from 'react';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Rockets from '../components/rockets';
import ShowRockets from '../components/showRockets';

describe('Rockets', () => {
  it('Rockets component is rendered correctly', () => {
    const rockets = render(
      <Provider store={store}>
        <Rockets />
      </Provider>,
    );
    expect(rockets).toMatchSnapshot();
  });

  it('ShowRockets component is rendered correctly', () => {
    const showRockets = render(
      <Provider store={store}>
        <ShowRockets />
      </Provider>,
    );
    expect(showRockets).toMatchSnapshot();
  });
});
