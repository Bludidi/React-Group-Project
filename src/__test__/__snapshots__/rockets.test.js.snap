// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should renders the  Component correctly 1`] = `
<div
  className="rocket-container"
>
  <div>
    <img
      alt="Rocket"
      className="rocket-img"
    />
  </div>
  <div
    className="rocket-info"
  >
    <h3 />
    <p>
      <span
        className="reserve-span"
        style={
          Object {
            "backgroundColor": "gray",
            "borderRadius": "0",
            "color": "#000",
            "marginRight": "0",
            "padding": "0",
          }
        }
      />
    </p>
    <button
      className="rocket-btn"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#4e4eff",
        }
      }
      type="button"
    >
      Reserve Rocket
    </button>
  </div>
</div>
`;
